plugins/gradle/gradle.plugin.sh

```diff
@@ -0,0 +1,24 @@
+#! bash oh-my-bash.module
+
+function gradle-or-gradlew() {
+  # find project root
+  # taken from https://github.com/gradle/gradle-completion
```

Comment:
That code seems to be licensed under the MIT license, meaning that [the original copyright notice](https://github.com/gradle/gradle-completion/blob/25da917cf5a88f3e58f05be3868a7b2748c8afe6/LICENSE#L1) should be retained. Please include it.